src/sentry/static/sentry/app/components/dropdownReact.jsx

```diff
@@ -160,6 +169,7 @@ class DropdownReact extends React.Component {
     let cx = classNames('dropdown-actor', className, {
       'dropdown-menu-right': isRight,
       'dropdown-toggle': true,
+      active: shouldShowDropdown,
```

Comment:
`active` on buttons causes it to be purple and light gray on hover:

![image](https://user-images.githubusercontent.com/79684/32873947-df164f5a-ca43-11e7-9604-98c1ac19fcae.png)